### ISSUE TITLE
Press enter to remove does not work

### ISSUE THREAD

**Issue by TopCoder1119** [orlyapps/nova-belongsto-depend#48]
Hi, When I tried to remove the selected option, it doesn't make it un-selected currently.

Can you please check if it works for you?

```
NovaBelongsToDepend::make('something')
                ->nullable()
                ->options(\something\Team::all())
                ->hideWhenCreating(),
```

**Comment by jimwhiteside** [comment]
I think I have encountered the same problem - if the field is defined as `nullable()` you can submit a field with no value selected, but once a value has been selected and saved you cannot then clear it.

Steps to reproduce:

1/ Create a record but leave this filed empty - model is created with null value
2/ Edit record and select a value, save, model is updated with selected value
3/ Edit record - clear value (delete key works for me), save, model retains the selected value

**Comment by orlyapps** [comment]
Thanks -working on a fix!

**Comment by orlyapps** [comment]
fixed in 9e6e8a4769813c593d50115968321e0d1f220ebb
